app/schemas/com.duckduckgo.app.global.db.AppDatabase/43.json

```diff
@@ -968,68 +968,12 @@
         },
         "indices": [],
         "foreignKeys": []
-      },
```

Comment:
The "downloads" table in "app.db" is now removed as it's moved to its own database in the `downloads-store` module.